generics/helpers/surveySubmissionUtils.js

```diff
@@ -71,6 +72,12 @@ const kafkaClient = require(ROOT_PATH + '/generics/helpers/kafkaCommunications')
             _.pick(surveySubmissionsDocument[0], ['project', 'status', '_id', 'completedDate'])
           );
         }
+
+        //adding question options, externalId to answers array 
```

Comment:
@Prajwal17Tunerlabs Please add more comments what happens in this function, should we rename the function if it is used not just for options?